runtime-builder/build.sh

```diff
@@ -26,12 +26,12 @@ usage() { echo "Usage: $0 <project> <go_version>"; exit 1; }
 base_digest()
 {
   # We sometimes have issue with 'describe', use the --log-http flag to get more info if it fails.
-  local digest="$(gcloud alpha container images describe gcr.io/google-appengine/debian8:latest | \
+  local digest="$(gcloud alpha container images describe gcr.io/distroless/base:latest | \
     grep '^Image:' | cut -d'@' -f2 | grep '^sha256:')"
 
   # The digest consists a prefix "sha256:", the hash string and a trailing newline character.
   if [[ "$(echo ${digest} | wc -c)" -ne 72 ]]; then
-    echo "$0: unable to parse digest of debian8 image: ${digest}"
+    echo "$0: unable to parse digest of base image: ${digest}"
```

Comment:
Mention distroless/base instead of just base since this script builds go1-builder image which doesn't use distroless/base image directly.